frontend/components/bc/BcMaintenanceBanner.vue

```diff
@@ -0,0 +1,47 @@
+<script setup lang="ts">
+import { warn } from 'vue'
+
+const { public: { maintenanceTS } } = useRuntimeConfig()
+const { tick } = useInterval(60)
+const { t: $t } = useI18n()
+
+const maintenanceLabel = computed(() => {
+  if (!maintenanceTS) {
+    return
+  }
+  const parsed = typeof maintenanceTS === 'number' ? maintenanceTS : parseInt(maintenanceTS)
+  if (isNaN(parsed)) {
+    warn('NUXT_PUBLIC_MAINTENANCE_TS is not convertable to an intager, a unix ts is expected')
+    return undefined
+  } else if (parsed === 0) {
```

Comment:
maybe `parsed <= 0`, so if Peter schedules a maintenance before 1970 we do not display the banner because we hope it is finished now.